packages/core/src/foundation.tsx

```diff
@@ -6,85 +6,101 @@
 
 import { NativeModules } from 'react-native';
 import { DdSdkConfiguration, DdSdkType, DdLogsType, DdTraceType, DdRumType } from './types';
+import {InternalLog} from "./InternalLog"
+import {SdkVerbosity} from "./SdkVerbosity";
 
 class DdLogsWrapper implements DdLogsType {
 
-  private nativeLogs: DdLogsType = NativeModules.DdLogs;
+    private nativeLogs: DdLogsType = NativeModules.DdLogs;
```

Comment:
This means that any log sent from the application will also go into the `console` if the verbosity is high enough ?

Reply:
Yes